test/e2e/specs/all-features-confirmation/footer/footer.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { test, expect } from '@playwright/test';
+
+// locators
+import PageLocators from '@isrd-isi-edu/chaise/test/e2e/locators/page';
+
+// utils
+import { getCatalogID } from '@isrd-isi-edu/chaise/test/e2e/utils/catalog-utils';
+
+const appName = ['recordedit', 'recordset'];
+test.describe('Page Footer', () => {
+    appName.forEach((name: string) => {
+
+        test.beforeEach(async ({ page, baseURL }, testInfo) => {
+            const PAGE_URL = `/${name}/#${getCatalogID(testInfo.project.name)}/product:accommodation`
+
+            await page.goto(`${baseURL}${PAGE_URL}`);
+        });
+
+        test('Checking footer in ' + name + ' page:', async ({ page }) => {
+            const footerMain = PageLocators.getFooterContainer(page);
+            const footerLink = PageLocators.getFooterLink(page);
+            
+            await test.step('footer should be visible on load', async () => {    
+                await footerMain.waitFor({ state: 'visible' });
```

Comment:
While this works and I think I actually done something like this in navbar tests, doing the following is better

```tsx
expect.soft(footerMain).toBeVisible();
```

I'm going to rewrite the `waitFor`s that I have in navbar specs to use expect instead.